src/develop/imageop.c

```diff
@@ -2332,10 +2332,20 @@ void add_remove_mask_indicator(dt_iop_module_t *module, gboolean add)
 
   if(module->mask_indicator)
   {
-    if(raster)
-      gtk_widget_set_tooltip_text(module->mask_indicator, _("this module has a raster mask"));
-    else
-      gtk_widget_set_tooltip_text(module->mask_indicator, _("this module has a mask, click to display\nmodule must be activated first"));
+    gchar *type;
+    gchar *tooltip;
+    const uint32_t mm = module->blend_params->mask_mode;
+    if((mm & DEVELOP_MASK_MASK) && (mm & DEVELOP_MASK_CONDITIONAL)) type=_("drawn + parametric mask");
+    else if(mm & DEVELOP_MASK_MASK) type=_("drawn mask");
+    else if(mm & DEVELOP_MASK_CONDITIONAL) type=_("parametric mask");
+    else if(mm & DEVELOP_MASK_RASTER) type=_("raster mask");
+    else dt_control_log("unknown mask mode '%d' in module '%s'", module->blend_params->mask_mode, module->op);
```

Comment:
this should not be reported on-screen but as all internal errors using  `fprintf(stderr, ...)`